docs/install/windows_setup.md

```diff
@@ -221,9 +222,6 @@ libgfortran-3.dll (in mingw64_dll)
 libquadmath-0.dll (in mingw64_dll)
 libopenblas.dll (in OpenBlas folder you download)
 opencv_world341.dll (in OpenCV folder you download)
-kernel32.dll
-vcomp140.dll
-nvcuda.dll (only if you use CUDA)
```

Comment:
Why do we remove these dlls?

Reply:
They are under C:Windows\System32 where python would check during loading libmxnet.dll